circuit_knitting/cutting/cutting_experiments.py

```diff
@@ -389,3 +370,80 @@ def _get_pauli_indices(cog: CommutingObservableGroup) -> list[int]:
     if not pauli_indices:
         pauli_indices = [0]
     return pauli_indices
+
+
+def _consolidate_resets(circuit: QuantumCircuit, inplace=True) -> QuantumCircuit:
```

Comment:
Slight improvement would be
```suggestion
def _consolidate_resets(circuit: QuantumCircuit, inplace: bool = True) -> QuantumCircuit:
```
and likewise for the other functions.  (maybe this will need black to run, too -- not sure)